sw/device/lib/testing/csrng_testutils.c

```diff
@@ -14,64 +14,69 @@ enum {
   kNumOutputWordsMax = 16,
 };
 
-void csrng_testutils_cmd_ready_wait(const dif_csrng_t *csrng) {
+status_t csrng_testutils_cmd_ready_wait(const dif_csrng_t *csrng) {
   dif_csrng_cmd_status_t cmd_status;
   do {
-    CHECK_DIF_OK(dif_csrng_get_cmd_interface_status(csrng, &cmd_status));
-    CHECK(cmd_status.kind != kDifCsrngCmdStatusError);
+    TRY(dif_csrng_get_cmd_interface_status(csrng, &cmd_status));
+    ASSERT_TRUE(cmd_status.kind != kDifCsrngCmdStatusError);
```

Comment:
again, updating `TRY` macro to accept a `bool` should remove the need for the `ASSERT_TRUE` macro. Applies throughout.

Reply:
done